index.html

```diff
@@ -0,0 +1,1532 @@
+              alt="image"
+            />
+            <div class="topic">
+              <a class="topilink" href="#"
+                >Don Jr. Set His Sights on Big Tech Months Before His Father's
+                Twitter Feud</a
+              >
+            </div>
+          </div>
+
+          <div class="col-12 col-md-4">
+            <img
+              src="https://d.newsweek.com/en/full/1595685/tuesday-primary-elections.webp?w=397&h=265&f=72276d64dab81e9c7d61d855dd436519"
+              alt="image"
+            />
+            <div class="topic">
+              <a class="topilink" href="#"
+                >What to Watch for As These 8 States and D.C. Head to the
+                Primaries</a
+              >
+            </div>
+          </div>
+
+          <div class="col-12 col-md-4">
+            <ul class="list-group">
+              <li class="list-group-item no-marg">
+                <img
+                  src="https://d.newsweek.com/en/full/1595769/elizabeth-warren.webp?w=95&h=95&f=5ddc35808c9708b7c89d93076aaf8b49"
+                  class="float-left list-img"
+                  alt="image"
+                />
+                <a class="no-marglink" href="#"
+                  >Elizabeth Warren Says Trump's Riot Plans Places 'Our
+                  Democracy in Danger'</a
+                >
+              </li>
+              <li class="list-group-item no-marg">
+                <img
+                  src="https://d.newsweek.com/en/full/1595769/elizabeth-warren.webp?w=95&h=95&f=5ddc35808c9708b7c89d93076aaf8b49"
+                  class="float-left list-img"
+                  alt="image"
+                />
+                <a class="no-marglink" href="#"
+                  >Elizabeth Warren Says Trump's Riot Plans Places 'Our
+                  Democracy in Danger'</a
+                >
+              </li>
+
+              <li class="list-group-item no-marg">
+                <img
+                  src="https://d.newsweek.com/en/full/1595756/tropical-depression-hurricane.webp?w=95&h=95&f=c7f642a3d869633aae58374d457e3e7d"
+                  class="float-left list-img"
+                  alt="image"
+                />
+                <a class="no-marglink" href="#"
+                  >Tropical Depression Forms in Gulf of Mexico on 1st Day of
+                  Hurricane Season</a
+                >
+              </li>
+            </ul>
+          </div>
+        </div>
+      </div>
+      <div class="row pre-foot">
+        <div class="col-12 col-lg-8 sub">
+          <img
+            src="https://d.newsweek.com/en/full/1589808/may-22-2020-cover.jpg?w=170&q=95&f=96101d0d430bef83aee5a95169c9b655"
+            alt="last one"
+            class="subimg1"
+          />
+          <img
+            src="https://d.newsweek.com/en/full/1591879/may-29-june-06-2020-cover.jpg?w=170&q=95&f=56aad51c81fc710a32b2778640b7e471"
+            alt="firstone"
+            class="subimg1"
+          />
```

Comment:
Make these two images overlaps over each other as on the original website.
![image](https://user-images.githubusercontent.com/34653764/83836062-5c573a80-a6ea-11ea-92cd-136131fba1f6.png)